README.md

```diff
@@ -58,16 +20,12 @@ If a lot of logs in a short period of time are marked as suspicious or anomalous
 The anomaly threshold is a number that can tuned depending on your volume of logs and how frequently Opni is predicting anomalies.
 ____
 #### Prerequisites
-TODO Production
 * At least two GPU nodes (K80 GPU or higher)
 * One node with at least 4 CPUs
 * At least three additional nodes each with at least 16 GB RAM and 40 GB of disk space.
 
-Want to try Opni out on on a lightweight cluster? (it's a lightweight opni with only control plane logs anomaly detection)
-TODO
 ____
 #### How does it work?
-TODO Move this to another markdown file
 Ship logs over to your Opni cluster with [Rancher Logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/). That's it! Opni will continuously learn the nature of your logs and will update models automatically.
 ____
```

Comment:
Can you replace this with a description of the updated method. Specifically remove `Ship logs over to your Opni cluster...`

`Opni will continuously learn the nature of your logs and will update models automatically.` can remain